index.html

```diff
@@ -0,0 +1,260 @@
+
+          <div class="card-text"><p>You will meet two McLaren's F1 world champions</p></div>
+
+          </div>  
+
+          <div class="program-card">
+
+            <div class="card-icon"><i class="fa-solid fa-file-contract"></i></div>
+
+            <div class="card-title"><h3>Analytics</h3></div>
+
+            <div class="card-text"><p>You will learn how to read and understand your race statistics</p></div>
+
+          </div>  
+
+          <div class="program-card">
+
+            <div class="card-icon"><i class="fa-solid fa-road"></i></div>
+
+            <div class="card-title"><h3>Free practice</h3></div>
+
+            <div class="card-text"><p>You will apply what you have learned with the help of our tutors</p></div>
+
+          </div>  
+
+          <div class="program-card">
+
+            <div class="card-icon"><i class="fa-solid fa-flag-checkered"></i></div>
+
+            <div class="card-title"><h3>Final race</h3></div>
+
+            <div class="card-text"><p>You will race against your partners and our F1 Champions</p></div>
+
+          </div>
+            
+        
+        </div>
+
+        <div class="program-buttonMobile"><h3>Join CC Global Summit 2015</h3> </div>
+
+        <div class="program-link"><a href="#program">SEE THE WHOLE PROGRAM</a></div>
+
+      </section>
+
+      <section id="speakers">
+
+        <div class="speakers-title"><h2>Featured speakers</h2></div>
+
+        <div class="speakers-separator"></div>
+
+        <div class="speakers-cardsContainer">
+
+          <div class="speakers-card" id="speakers-card0">
+
+            <div class="spk-img"><img src="#" id="spk-img0" alt="#"></div>
+
+            <div class="spk-textContainer">
+              <div class="spk-title" id="spk-title0"></div>
+              <div class="spk-subTitle" id="spk-subTitle0"></div>
+              <div class="spk-text" id="spk-text0"></div>
+            </div>
+
+          </div>
+
+          <div class="speakers-card" id="speakers-card1">
+
+            <div class="spk-img">
+              <img src="#" id="spk-img1" alt="#">
+            </div>
+
+            <div class="spk-textContainer">
+              <div class="spk-title" id="spk-title1"></div>
+              <div class="spk-subTitle" id="spk-subTitle1"></div>
+              <div class="spk-text" id="spk-text1"></div>
+            </div>
+
+          </div>
+
+          <div class="speakers-card" id="speakers-card2">
+
+            <div class="spk-img"><img src="#" id="spk-img2" alt="#"></div>
+
+            <div class="spk-textContainer">
+              <div class="spk-title" id="spk-title2"></div>
+              <div class="spk-subTitle" id="spk-subTitle2"></div>
+              <div class="spk-text" id="spk-text2"></div>
+            </div>
+
+          </div>
+
+          <div class="speakers-card" id="speakers-card3">
+
+            <div class="spk-img"><img src="#" id="spk-img3" alt="#"></div>
```

Comment:
- [ ] Please use a loop to create the speakers' section dynamically. You should create every element related to this section in the loop in your `.js` file.